contracts/bridge/MiniChefV21.sol

```diff
@@ -26,32 +29,32 @@ contract MiniChefV21 is BoringOwnable, BoringBatchable {
     /// @notice Info of each MCV2 user.
     /// `amount` LP token amount the user has provided.
     /// `rewardDebt` The amount of SYNAPSE entitled to the user.
-    struct UserInfo {
-        uint256 amount;
-        int256 rewardDebt;
-    }
+    //    struct UserInfo {
```

Comment:
Lets ditch thesse and move these comments to the interface

Reply:
[3b02b24](https://github.com/synapsecns/synapse-contracts/pull/90/commits/3b02b24b51c469114c98d6165e79ceec9fd940b3)